### PR TITLE
add first html structure + content

### PR DESCRIPTION
**Pull request by johnxgaehner**

### CODE REVIEW

**Review by adrianammt — APPROVED** [review]
Nice structure.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    <title>John Goehner</title>
+  </head>
+  <body>
+    <header>
+      <h1>JOHN GOEHNER</h1>
+      <p>ride fast - take chances!</p>
+      <nav>
+        <a href="#am-section">About Me</a>
+        <a href="#cv-section">Curriculum Vitae</a>
+        <a href="#lc-section">Let's Connect!</a>
```
Comment (mpagels) [inline index.html:16]: You could put the anchors inside a list.

See examples here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#examples

File: index.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      </nav>
+    </header>
+
+    <main>
```
Comment (mpagels) [inline index.html:20]: Very good to put the main content in the `main` element. Very semantic. I like that.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        src="images/red-hook-crit.jpg"
+        alt="John, riding a fixed gear bike"
+      />
+      <p>
+        picture taken by
+        <a href="https://www.chiararedaschi.com/">Chiara Redaschi</a>
+      </p>
+
+      <section id="am-section">
+        <h2>About Me</h2>
```
Comment (mpagels) [inline index.html:31]: Very good to start here with a `h2` element.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+          When I ride my bike, racing is always on my mind. It’s all about
+          flying through high speed corners, pushing big gears on long
+          straights, attacking out of nowhere. My favourite group ride?
+          Partyteller!
+        </p>
+      </section>
+
+      <section id="cv-section">
+        <h2>Curriculum Vitae</h2>
+
```
Comment (mpagels) [inline index.html:49]:
```suggestion
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+          <li>training km: 6.500km</li>
+          <li>training time: 250h</li>
+          <li>races: 01 / wins: 00</li>
+        </ul>
+      </section>
+
+      <section id="lc-section">
+        <h2>Let's Connect!</h2>
+        <a href="https://www.instagram.com/johnxgaehner/">Instagram</a>
+        <a href="https://www.strava.com/athletes/6047784">Strava</a>
```
Comment (mpagels) [inline index.html:94]: A list would be helpful here as well.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+      </section>
+    </main>
+
+    <footer>
+      <p>
+        John Goehner <br />
+        <a href="mailto:hello@test.com">hello@test.com</a> <br />
+        <a href="tel:+499123456789">+49 177 123 45 67</a></p>
```
Comment (mpagels) [inline index.html:102]: And here another list.

**Review by mpagels — APPROVED** [review]
Very good work, John. I found some little things. Please check my comments.

**Pull request merged**